U8HF15/AI_Practice/CommandFiles/Analytics.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AI_Practice.CommandFiles
+{
+    public class Analytics : CommandInterface
+    {
+        public string Name => "analytics";
+
+        public string Description => "információt add vissza a dataset-ről";
+
+        public string LongDescription => "Analízálja a dataset-et és információkat közöl róla";
+
+        public void Run(object[]? args)
+        {
+            Network ai = Commands.AI;
+
+            List<float[]> outputs = new();
+            for (int j= 0,i=0; j < ai.Dataset[0].Outputs.Length; j++){
+
+                float[] output = ai.Dataset.SelectMany(f => f.Outputs).Where(f => (i++)%ai.Dataset[0].Outputs.Length == j).ToArray();
+                outputs.Add(output);
+            }
+
+            var maxes = outputs.Select(f=> f.Max()).ToList();
```

Comment:
Ez így nem a legoptimálisabb. Maximum, Minimum és Átlag számolás ebben a formában 3x iterál végig az adathalmazon. Ezt meg lehetne csinálni egy iterációban is.